system/jlib/jptree.cpp

```diff
@@ -7881,11 +7879,13 @@ jlib_decl IPropertyTree * loadConfiguration(const char * defaultYaml, const char
         componentDefault.set(defaultConfig->queryPropTree(componentTag));
         if (!componentDefault)
             throw makeStringExceptionV(99, "Default configuration does not contain the tag %s", componentTag);
+        if (*defaultYaml=='{')
+            componentDefault.setown(mapJsonToXmlConfig(componentDefault));
     }
     else
         componentDefault.setown(createPTree(componentTag));
 
-    Owned<IPropertyTree> config(mapJsonToXmlConfig(componentDefault));
+    Owned<IPropertyTree> config(componentDefault);
```

Comment:
This should be Linked rather than Owned, since it isn't taking ownership.  (I think for some strange reason this code may  work with Owned.)